_posts/2021-04-20-jump_dev_2021.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title:  "JuMP-dev 2021"
+date:   2021-04-20
+categories: [jump-dev]
+permalink: /meetings/juliacon2021/
+---
+
+We are pleased to announce that JuMP-dev 2021 will run as a separate track of
+[JuliaCon 2021](https://juliacon.org/2021/)!
```

Comment:
Is this the wording the JuliaCon committee suggested? I thought JuMP-dev would be officially co-located with JuliaCon rather than being a track of JuliaCon.

Reply:
I can change the wording. Avik didn't mind, and will point to this post when he updates the JuliaCon website.

Reply:
How's this.